modules/seqexec/web/client/src/main/scala/seqexec/web/client/SeqexecApp.scala

```diff
@@ -31,6 +29,9 @@ final class SeqexecLauncher[F[_]](implicit val F: Sync[F], L: LiftIO[F]) {
     setLoggerThreshold("", AllThreshold)
   }
 
+  private implicit val ioContextShift: ContextShift[IO] =
```

Comment:
Why didn't we need this before?

Reply:
Good question, I will look more closely.

Reply:
`IO.fromFuture` now takes `ContextShift[IO]`. Daniel added late in the release cycle https://github.com/typelevel/cats-effect/commit/ebab1db7b9a116c939c60213339743d5b9907ac9